helpers/getEditUrl.js

```diff
@@ -1,5 +1,5 @@
 module.exports = function() {
   var msPath = arguments[0];
 
-  return 'https://github.com/trufflesuite/trufflesuite.com/edit/master/src/' + msPath.dir;
+  return 'https://github.com/trufflesuite/trufflesuite.com/edit/develop/src/' + msPath.dir;
 };
```

Comment:
Could you put a newline here also?

Reply:
drive by improvements are not the name of my game, but if that'll get it approved

Reply:
haha I have one other comment